proto/executor.proto

```diff
@@ -145,6 +149,12 @@ message ANNQueryInfo {
 	optional double max_distance = 10 [(gogoproto.nullable) = false]; // Only for ANNQueryType==Where
 
 	optional uint32 hnsw_ef_search = 20 [(gogoproto.nullable) = false]; // Only for HNSW indexes
+
+	// If enabled, the content of TableScan's vector output column (whose ID is column_id) will not be
+	// used and TableScan must output an additional column using the Distance function (whose id must be -2000)
+	// Implementor does not need to really generate the
+	// Vector data column.
```

Comment:
This comment is outdated. A more correct one is that, if enabled, TableScan's schema will not include Vector column at all, .... etc?